site/content/contribute/getting-started/core-committers.md

```diff
@@ -59,6 +59,9 @@ Below is the list of core committers working on Mattermost:
 - **Brad Coughlin**
     - @brad on [community.mattermost.com](https://community.mattermost.com/core/messages/@brad) and [@bradjcoughlin](https://github.com/bradjcoughlin) on GitHub
     - Dev areas: Web App, React, Redux
+- **Lev Brouk**
```

Comment:
nit: I think it's worthwhile to maintain the "order" here, so I'd locate you after Hanzei. We also need @mgdelacroix?